src/main/java/org/takes/rq/RqForm.java

```diff
@@ -167,10 +168,12 @@ private static String decode(final CharSequence txt) {
                     final String body = new String(baos.toByteArray());
                     final ConcurrentMap<String, List<String>> map =
                         new ConcurrentHashMap<String, List<String>>(1);
+                    // @checkstyle MultipleStringLiteralsCheck (1 line)
                     for (final String pair : body.split("&")) {
                         if (pair.isEmpty()) {
                             continue;
                         }
+                        // @checkstyle MultipleStringLiteralsCheck (1 line)
```

Comment:
@erimerturk We can extend the previous warning to 5 lines instead of having a separate one here.